dist/c-jwt.spec

```diff
@@ -0,0 +1,62 @@
+Summary:            JSON Web Token (C implementation)
+Name:               c-jwt
+Version:            1.8.0
```

Comment:
I suggest making this a `libjwt.spec.in` file and have autoconf fill in the correct values (i.e. Version).

Reply:
this implies to run ./configure before rpmbuild -ba dist/libjwt.spec.
Is that what you want ?

Reply:
Well, you have two alternatives. The Makefile.am route and generate Makefile with ./configure, or Makefile that parses the version from configure.in. Both would create the spec file from a .in with the version number.